wrapanapi/systems/google.py

```diff
@@ -101,16 +101,24 @@ def _get_state(self):
         return self._api_state_to_vmstate(self.raw['status'])
 
     @property
-    def ip(self):
+    def ip_internal(self):
```

Comment:
We need to make sure we update integration_tests so this doesn't break anything.

Reply:
YOLO

Reply:
On a more serious note - yep, I'm making changes to integration_tests and will have a PR there for changes and wrapanapi version update.

Reply:
do you think we need it ? I guess presence of it in all_ips would be enough

Reply:
Not really, but wanted to provide a method for it in case something/one _was_ actually using it directly.